ui/lib/core/addon/components/linkable-item.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @module LinkableItem
+ * LinkableItem components have two contextual components, a Content component used to show information on the left with a Menu component on the right, all aligned vertically centered. If passed a link, the block will be clickable.
+ *
+ * @example
+ * ```js
+ * <LinkableItem @link={{hash route='vault.backends' model='my-backend-path'}} data-test-row="my-backend-path">
+ * // Use <LinkableItem.content> and <LinkableItem.menu> here
```

Comment:
Contextual components would not display in storybook, so added as comments in notes file.